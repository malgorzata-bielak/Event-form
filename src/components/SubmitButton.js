import React from "react";

const SubmitButton = () => (
  <>
    <button>Submit</button>
  </>
);

export default SubmitButton;
