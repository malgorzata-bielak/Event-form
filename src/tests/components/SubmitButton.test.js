import React from "react";
import { shallow } from "enzyme";

import SubmitButton from "../../components/SubmitButton";

test("should render SubmitButton correctly", () => {
  const wrapper = shallow(<SubmitButton />);
  expect(wrapper).toMatchSnapshot();
});
