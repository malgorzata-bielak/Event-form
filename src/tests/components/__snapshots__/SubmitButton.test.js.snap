// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render SubmitButton correctly 1`] = `
<Fragment>
  <Styled(button)>
    Submit
  </Styled(button)>
</Fragment>
`;
